internal/services/object/helpers_object.go

```diff
@@ -47,8 +49,25 @@ func (r *scalewayResolver) ResolveEndpoint(ctx context.Context, params s3.Endpoi
 	return s3.NewDefaultEndpointResolverV2().ResolveEndpoint(ctx, params)
 }
 
+type userAgentRoundTripper struct {
+	defaultRoundTripper http.RoundTripper
+	userAgent           string
+}
+
+var userAgent = fmt.Sprintf("scaleway-terraform-provider/%s (%s; %s; %s)", defaultVersion, runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
```suggestion
var userAgent = fmt.Sprintf("terraform-provider/%s (%s; %s; %s)", defaultVersion, runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Reply:
You could also add the version of terraform as well. 
here is an example of what we currently have:
```
scaleway-sdk-go/v1.0.0-beta.30.0.20241129094524-023aa8142bc1 (go1.23.1; linux; amd64) terraform-provider/2.49.0 terraform/1.10.4
```